assets/dev/js/editor/controls/media.js

```diff
@@ -50,10 +53,12 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 
 		if ( [ 'image', 'svg' ].includes( mediaType ) ) {
 			this.ui.mediaImage.css( 'background-image', url ? 'url(' + url + ')' : '' );
-
 			if ( ! value && url ) {
 				this.ui.mediaImage.css( 'opacity', 0.5 );
 			}
+			this.ui.mediaImage.attr( 'data-id', this.getControlValue( 'id' ) || this.getControlPlaceholder()?.id );
+			this.ui.mediaImage.attr( 'data-alt', this.getControlValue( 'alt' ) || this.getControlPlaceholder()?.alt || '' );
+			this.ui.mediaImage.attr( 'data-size', this.getControlValue( 'size' ) || this.getControlPlaceholder()?.size || 'full' );
```

Comment:
explain and/or move to a meaningful function

Reply:
Done